pipelines/visitQualityCore.yaml

```diff
@@ -43,3 +43,9 @@ tasks:
     config:
       atools.astromColorDiffMetrics: TargetRefCatDeltaColorMetrics
       python: from lsst.analysis.tools.atools import *
+  relativeSizeResidualPlot:
+    class: lsst.analysis.tools.tasks.SourceTableVisitAnalysisTask
+    config:
+      connections.outputName: PSFResidualPlot
+      atools.relativeSizeResidualPlot: RelativeSizeResidualPlot
+      python: from lsst.analysis.tools.atools import *
```

Comment:
Move line 50 to a new line between lines 11 and 12 and you can drop everything else here.